create_certs.sh

```diff
@@ -1,7 +1,7 @@
 #! /bin/bash
-
-openssl genrsa -des3 -passout pass:x -out server.pass.key 2048
-openssl rsa -passin pass:x -in server.pass.key -out server.key
+PASS=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
Just curious - is this a random password generator?

Reply:
Yup.